web/src/lib/domain-logic/filterCertifications.ts

```diff
@@ -1,16 +1,27 @@
 import { Certification } from "@/lib/types/types";
 import { UserData } from "@businessnjgovnavigator/shared";
 
+export const SMALL_BUSINESS_MAX_EMPLOYEES = 119;
```

Comment:
If the logic in the story is "lower than 120", it feels weird to use 119 here. can we use >= 120 instead of > 119, because 120 is the magic-number called out in the story

Reply:
@Jesslax makes sense to me if we use >=120